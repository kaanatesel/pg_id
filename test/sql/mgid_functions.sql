SELECT mgid_get_timestamp('00000000bcf86cd799439011');
SELECT mgid_get_timestamp('00000000810c19729de860ea');

SELECT mgid_get_process_unique('507f1f770000000000439011');
SELECT mgid_get_process_unique('507f191d0000000000e860ea');

SELECT mgid_get_counter('507f1f77bcf86cd799000000');
SELECT mgid_get_counter('507f191e810c19729d000000');