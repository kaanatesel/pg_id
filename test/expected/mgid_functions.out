SELECT mgid_get_timestamp('00000000bcf86cd799439011');
 mgid_get_timestamp 
--------------------
 00000000
(1 row)

SELECT mgid_get_timestamp('00000000810c19729de860ea');
 mgid_get_timestamp 
--------------------
 00000000
(1 row)

SELECT mgid_get_process_unique('507f1f770000000000439011');
 mgid_get_process_unique 
-------------------------
 0000000000
(1 row)

SELECT mgid_get_process_unique('507f191d0000000000e860ea');
 mgid_get_process_unique 
-------------------------
 0000000000
(1 row)

SELECT mgid_get_counter('507f1f77bcf86cd799000000');
 mgid_get_counter 
------------------
 000000
(1 row)

SELECT mgid_get_counter('507f191e810c19729d000000');
 mgid_get_counter 
------------------
 000000
(1 row)

